### PR TITLE
[frontend] List of opinions have UI issues (#9186)

### PR DESCRIPTION
**Pull request by SarahBocognano**
### Proposed changes

* Several changes have been made here cause Opinion had quit a lot of issues
* Refactoring OpinionPopover in tsx
* Fix style to resolve opinion list UI issues and add a tooltip for long descriptions
* Fix the "auto open" of list when adding an opinion
* Fix the message error if non opinion who wasn't triggered when no opinion
* Fix the fetching of data when deleting an opinion (the radar didn't update without refreshing the page)

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/9186
* https://github.com/OpenCTI-Platform/opencti/issues/9091

### CODE REVIEW

File: opencti-platform/opencti-front/lang/front/fr.json [hunk 1]
```diff
@@ -2661,6 +2661,7 @@
   "The min value cannot be greater than max value": "La valeur min ne peut pas être supérieure à la valeur max",
   "The operators and modes are restricted for these filters.": "Les opérateurs et les modes sont restreints pour ces filtres.",
   "The opinions has no value defined in your vocabulary. Please add them first to be able to add opinions.": "Les opinions n'ont pas de valeur définie dans votre vocabulaire. Veuillez d'abord les ajouter pour pouvoir ajouter des opinions.",
+  "The opinions have no value defined in your vocabulary. Please add them first to be able to add opinions.": "Les avis n'ont pas de valeur définie dans votre vocabulaire. Veuillez d'abord les ajouter pour pouvoir ajouter des opinions.",
```
Comment (Archidoit) [inline opencti-platform/opencti-front/lang/front/fr.json]:
```suggestion
  "The opinions have no value defined in your vocabulary. Please add them first to be able to add opinions.": "Les opinions n'ont pas de valeur définie dans votre vocabulaire. Veuillez d'abord les ajouter pour pouvoir ajouter des opinions.",
```

File: opencti-platform/opencti-front/lang/front/fr.json [hunk 2]
```diff
@@ -2661,6 +2661,7 @@
   "The min value cannot be greater than max value": "La valeur min ne peut pas être supérieure à la valeur max",
   "The operators and modes are restricted for these filters.": "Les opérateurs et les modes sont restreints pour ces filtres.",
   "The opinions has no value defined in your vocabulary. Please add them first to be able to add opinions.": "Les opinions n'ont pas de valeur définie dans votre vocabulaire. Veuillez d'abord les ajouter pour pouvoir ajouter des opinions.",
```
Comment (Archidoit) [inline opencti-platform/opencti-front/lang/front/fr.json]: useless translation

**Comment by Archidoit** [comment]
I have no opinions, when I click on the radar the error messages display : 
image

Whereas it's false, I have opinions in my vocabulary.
The message should rather be : 'No opinions have been added for this entity.'. OR display an empty opinions list


@SarahBocognano : Done

**Comment by Archidoit** [comment]
Markings are not aligned in the opinions list : 
image

**Comment by CelineSebe** [comment]
Why is it possible to delete an opinion directly, but not to update it? This give us the feeling, it forbidden
image


@SarahBocognano : You can edit an opinion by clicking on the thumbs. If you already added one, the form will be an update. Seen with product: Don't need to have an update here

**Comment by CelineSebe** [comment]
A display issue on a smaller screen:
image


I don't have this issue, exemple with an Iphone screen size : 
image

**Comment by CelineSebe** [comment]
tested locally 👌

**Review by CelineSebe — APPROVED** [review]

**Pull request merged**